opt/definitions.rs

```diff
@@ -143,6 +143,20 @@ pub struct Dominators {
     /// The maximum number of rows, regardless of depth in the tree, to display.
     #[structopt(short = "r")]
     max_rows: Option<u32>,
+
+    /// Whether or not `items` should be treated as regular expressions.
+    #[structopt(long = "regex")]
+    using_regexps: bool,
+}
+
+impl Dominators {
+    // TODO: wasm-bindgen does not support sending Option<String> across
+    // the wasm ABI boundary yet.
```

Comment:
It is unclear to me how this TODO is relevant? Is this copy-pasted?

Reply:
Good eye! This comment was mimicking what I saw in the implementation for `Paths` in that file. The comment should say: `// TODO: wasm-bindgen does not support sending Vec<String> across the wasm ABI boundary yet.`

I refactored the `subtree` member in this file from an `Option<String>` into a `Vec<String>` so that it could support multiple arguments in this commit. This was definitely a copy paste error :grimacing:

I fixed this and squashed it into the commit, so it should be good to go now :)